doc/faq/manual.rst

```diff
@@ -80,31 +80,31 @@ Does anybody actually use Charm++?
 
 Several large applications use Charm++.
 
-- The large, production-quality molecular dynamics application `NAMD <http://charm.cs.illinois.edu/research/moldyn/>`_.
-- The cosmological simulator `ChaNGa <http://charm.cs.illinois.edu/research/cosmology/>`_.
-- The atomistic simulation framework `OpenAtom <http://charm.cs.illinois.edu/OpenAtom/>`_.
+- The large, production-quality molecular dynamics application `NAMD <https://charm.cs.illinois.edu/research/moldyn/>`_.
+- The cosmological simulator `ChaNGa <https://charm.cs.illinois.edu/research/cosmology/>`_.
+- The atomistic simulation framework `OpenAtom <https://charm.cs.illinois.edu/OpenAtom/>`_.
 - We have significant collaborations with groups in Materials Science,
   Chemistry, Astrophysics, Network Simulation, Operation Research,
   Contagion Effects, in Illinois, New York, California, Washington, and
-  Virginia. See also `Applications <http://hpccharm.com/products.html#applications>`_ for a more complete list.
+  Virginia. See also `Applications <https://hpccharm.com/products.html#applications>`_ for a more complete list.
```

Comment:
@stwhite91 @evan-charmworks This URL doesn't work with http or https.

Reply:
```suggestion
  Virginia. See also `Applications <https://hpccharm.com/applications>`_ for a more complete list.
```